### ISSUE TITLE
Pre-6.0 builds should also be accessible using the new aka.ms link infrastructure

### ISSUE THREAD

**Issue by adiaaida** [dotnet/arcade#6853]
5.x and 3.x builds should also be able to be accessible using the aka.ms links generated.

**Comment by adiaaida** [comment]
This will work automatically with 5.X links going forward. 3.X needs to be moved to publishing v3 to get the links for signed, daily, and validated, but will get ga automatically.

**Comment by mmitche** [comment]
> However, as all 3.x builds are signed, I don't know if I see the ROI for this work (daily and signed links would be identical). @mmitche thoughts on this?

Sounds like it could be skipped, as long as it doesn't negatively impact dotnet-install too much.

**Comment by adiaaida** [comment]
@bozturkMSFT thoughts on 3.x signed vs daily?

**Comment by bozturkMSFT** [comment]
I don't fully understand the question here, but I will still try to answer hoping it will help the discussion here until you can clarify it for me.

Is the question here: Should we start using aka.ms links with 3.x?
- I don't think it matters much from install scripts perspective. If the final answer is "no", then we should make sure 3.x builds always have latest.version files.

Is the question here: Should we generate aka.ms links for daily 3.x builds, even though the downloaded artifacts will be identical to the signed ones?
- My vote would be yes. I don't think users will complain that the daily builds are signed. But if we don't allow `-quality daily` with 3.x, then we will need to explain this in the docs. Unaware of the docs, some people will still try to do it and create issues in our repo.

**Comment by adiaaida** [comment]
I think the question was more, should we add publishing to the servicing pipeline for 3.x so that we have artifacts of the "signed" quality. When 3.x is converted to publishing v3, it will start publishing -quality daily by default. So what I'm asking is, do we also want -quality signed? We will also probably need to do extra work for -quality validated.

**Comment by bozturkMSFT** [comment]
In short: if there is nothing standing in the way, yes.

 I think we should either be doing this the old way or the new way, but not somewhere in-between. If we kept 3.x working exactly the same (with latest.version files), this would be fine. However, *if we are breaking the old way*, then we should fully switch to the new way. This will otherwise cause confusion and will return to us as bug reports. 

 Even us as developers having a hard time understanding what behavior is a bug and what behavior is expected. I wouldn't want to have to learn yet another condition like "... if the version is 3.x, only x works but not y and z...".

**Comment by adiaaida** [comment]
That makes sense. I am going to move this to the Release v2 epic, since it relies on other work that isn't complete. I will prioritize it once 3.x is on publishing v3.

**Comment by adiaaida** [comment]
As this is blocked by the 3.x V3 conversion, I believe this to be out of scope for #6418.
